src/WinRT.Runtime/Interop/ExceptionErrorInfo.cs

```diff
@@ -2,49 +2,17 @@
 // Licensed under the MIT License.
 
 // This file was generated by cswinrt.exe
+
 using System;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
 namespace WinRT.Interop
-{
-    [Guid("1CF2B120-547D-101B-8E65-08002B2BD119")]
-    internal interface IErrorInfo
-    {
-        Guid GetGuid();
-        string GetSource();
-        string GetDescription();
-        string GetHelpFile();
-        string GetHelpFileContent();
-    }
-
-    [Guid("DF0B3D60-548F-101B-8E65-08002B2BD119")]
-    internal interface ISupportErrorInfo
-    {
-        bool InterfaceSupportsErrorInfo(Guid riid);
-    }
-
-    [Guid("04a2dbf3-df83-116c-0946-0812abf6e07d")]
-    internal interface ILanguageExceptionErrorInfo
-    {
-        IObjectReference GetLanguageException();
-    }
-
-    [Guid("82BA7092-4C88-427D-A7BC-16DD93FEB67E")]
-    internal interface IRestrictedErrorInfo
-    {
-        void GetErrorDetails(
-            out string description,
-            out int error,
-            out string restrictedDescription,
-            out string capabilitySid);
-
-        string GetReference();
-    }
-
+{
 #if NET
     [WinRTExposedType(typeof(ManagedExceptionErrorInfoTypeDetails))]
 #endif
-    internal sealed class ManagedExceptionErrorInfo : IErrorInfo, ISupportErrorInfo
+    internal sealed class ManagedExceptionErrorInfo
```

Comment:
Are we no longer projecting these interfaces on the vtable for net standard?

Reply:
No, I'm just always using `IUnknownVftbl` and explicitly passing the IID of the error interface to use. All those interfaces were internal anyway and looking at the code it didn't look like the NS2.0 paths were really relying on any of those vtable bits either. All tests are passing too on all targets, so I guess that assumption was correct? 😅

Reply:
The `ManagedExceptionErrorInfoTypeDetails` where the IIDs are explicitly passed is not today used on .NET Standard.  The following is in theory where this would be used in but only happens on down-level windows versions: https://github.com/microsoft/CsWinRT/blob/2b77d8412adb7b62bf8f1265a6d8b8c3169dde58/src/WinRT.Runtime/ExceptionHelpers.cs#L354

Reply:
Mmh right. I'm not sure I understand how are all the tests still passing then (are we missing some coverage here?), but yeah I'll double check that path and revert some downlevel changes if needed then 👍

Reply:
Ok I've reverted all downlevel changes and added a separate file for .NET 6+ with that simplified API.